JS/Marquee.js

```diff
@@ -0,0 +1,49 @@
+class Marquee {
```

Comment:
nice implementation of the marquee as a class!
please note that in the UI (browser), the Marquee is making the width of the page very wide. you should prevent the marquee from widing the page (in css)